bin/diagnostic.js

```diff
@@ -1 +1,13 @@
 'use strict';
+
+const ask = require('../lib/ask.js');
+
+let answer = 'not empty';
+
+while (answer !== '7') {
+  answer = ask("Guess a number between 1 and 10 ");
+  if (answer === '7') {
+   console.log("You guessed it! You should play the Powerball");
+   break;
+  }
+}
```

Comment:
First your checking to see if answer is === or !== to ```'7'``` which is a string, it should just be ```7```. Also you forgot to ask if the number is between 1 and 10. You can check by writing either ```( answer < 1 || answer > 10)``` to check if the answer falls outside that range OR ```(answer >= 1 && answer <= 10)``` to see if answer falls inside that range.